packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -519,6 +537,7 @@ export function terminateAllRunningTeamsfxTasks(): void {
   }
   allRunningTeamsfxTasks.clear();
   BaseTunnelTaskTerminal.stopAll();
+  void deleteAad();
```

Comment:
Can you share whether this also apply to Python TTK projects? We can discuss more if you need to learn more details for Python project.